src/index.ts

```diff
@@ -8,30 +8,28 @@ const movieDisplay = document.getElementById(
 const movieSearchBox = document.getElementById(
   'movie-search-box'
 ) as HTMLInputElement;
-const searchList = document.getElementById('search-list') as HTMLInputElement;
-let searchTerm = movieSearchBox.value.trim();
 
 // Get Search Term to call API
 function searchMovies(searchTerm: string): void {
   if (searchTerm.length > 0) {
     loadMovies(searchTerm);
   }
 }
-// API Call
+// Fetch Data, Return JSON
```

Comment:
If you're looking for structure in your comments, you can consider following a style guide like Google's: https://google.github.io/styleguide/tsguide.html#comments-documentation. The benefit of that is consistency. It's not critical in a project with only a handful of functions with a single developer though, so it's up to you if you want to put in the additional effort (too many comments can be a drawback because they just end up requiring a lot of work to keep in sync as you refactor code over time). The style guide has other useful info too if you're not sure about random stuff like when to use `===` or `==` https://google.github.io/styleguide/tsguide.html#equality-checks.

Reply:
it's just guidance tho, don't study the whole thing or get slowed down with it. It's more important to build stuff and learn style along the way.

Reply:
Thanks for this resource, I'll definitely start referring to and implementing this going forward. I'll also look into seeing if I can get the linting to work for TypeScript as well.